xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -390,8 +390,6 @@ namespace XBMCAddon
             item->GetVideoInfoTag()->m_strProductionCode = value;
           else if (key == "aired")
             item->GetVideoInfoTag()->m_firstAired.SetFromDateString(value);
-          else if (key == "credits")
-            item->GetVideoInfoTag()->m_writingCredits = StringUtils::Split(value, g_advancedSettings.m_videoItemSeparator);
```

Comment:
I guess I missed that. Remind me why backward compatibility wasn't kept? It seem like it's only a rename, in which case keeping backward compatibility would have been easy..

Even if 'credits' it isn't used it's *more work* to find that out than to keep backward compatibility.. which is do nothing.